tests/test_training_validation.py

```
@@ -0,0 +1,681 @@
+    }
+
+
+def _get_training_args_field_names():
+    """Return all field names defined on the custom TrainingArguments dataclass."""
+    return {f.name for f in dataclass_fields(TrainingArguments)}
+
+
+def _make_training_args(**overrides):
+    """Create a TrainingArguments with a temporary output_dir and no reporting."""
+    defaults = {
+        "output_dir": "/tmp/gliner_test_output",
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/gliner_test_output"

(S108)